pkgs/tools/graphics/agi/default.nix

```diff
@@ -9,63 +9,69 @@
 , gobject-introspection
 , gdk-pixbuf
 , jre
-, androidenv
+, android-tools
 }:
 
-stdenv.mkDerivation rec {
+stdenvNoCC.mkDerivation rec {
   pname = "agi";
-  version = "3.2.0-dev-20220831";
+  version = "3.0.1";
 
   src = fetchzip {
-    url = "https://github.com/google/agi-dev-releases/releases/download/v${version}/agi-${version}-linux.zip";
-    sha256 = "sha256-pAPYIhNqr7TpVDnHhRVGkd6HCqu945OCXa5dpGi2UhU=";
+    url = "https://github.com/google/agi/releases/download/v${version}/agi-${version}-linux.zip";
+    sha256 = "sha256-793lOJL1/wqETkWfiksnLY3Lmxx500fw4PIzT9ZQqQs=";
   };
 
   nativeBuildInputs = [
-    autoPatchelfHook
-    makeWrapper
     wrapGAppsHook
     gdk-pixbuf
     gobject-introspection
+    autoPatchelfHook
     copyDesktopItems
-  ];
-
-  buildInputs = [
-    stdenv.cc.cc.lib
+    makeWrapper
   ];
 
   installPhase = ''
     runHook preInstall
-    mkdir -p $out/{bin,lib}
+
+    mkdir -p $out/bin
     cp ./{agi,gapis,gapir,gapit,device-info} $out/bin
-    cp lib/gapic.jar $out/lib
-    wrapProgram $out/bin/agi \
-      --add-flags "--vm ${jre}/bin/java" \
-      --add-flags "--jar $out/lib/gapic.jar" \
-      --add-flags "--adb ${androidenv.androidPkgs_9_0.platform-tools}/bin/adb"
+    cp -r lib $out
+
     for i in 16 32 48 64 96 128 256 512 1024; do
-      install -D ${src}/icon.png $out/share/icons/hicolor/''${i}x$i/apps/agi.png
+      install -D ${src}/icon.png $out/share/icons/hicolor/$ix$i/apps/agi.png
```

Comment:
Is this intentional? Icons are likely now installed in e.g. `$out/share/icons/hicolor/16/apps` instead of `$out/share/icons/hicolor/16x16/apps`

Reply:
It wasn't, I must've done that by accident

Reply:
Opened a PR which fixes it: https://github.com/NixOS/nixpkgs/pull/191488